src/treeView.ts

```diff
@@ -184,7 +188,24 @@ export class TreeDataProvider implements vscode.TreeDataProvider<TreeItem> {
       let functionalTestsItem = this.data.find((item) => item.label && item.label.toString().startsWith("Functional"));
       let buidUseCaseItem = this.data.find((item) => item.label && item.label.toString().startsWith("Build"));
       let selectVariantItem = this.data.find((item) => item.label && item.label.toString().startsWith("Select variant"));
-
+      let checkItem = this.data.find((item) => item.label && item.label.toString().startsWith("Run Guideline Enforcer"));
+
+      this.data.forEach((item) => {
+        if (item.label && item.label.toString().startsWith("Docker Container")) {
+          let terminalItem = item.children?.find((child) => child.label && child.label.toString().startsWith("Open terminal"));
+          if (terminalItem) {
+            const conf = vscode.workspace.getConfiguration("ledgerDevTools");
+            if (conf.get<boolean>("openContainerAsRoot") === false) {
+              terminalItem.label = `Open terminal`;
+            } else {
+              terminalItem.label = `Open terminal [root]`;
+            }
+          }
+        }
+      });
+      if (checkItem) {
+        checkItem.label = `Run Guideline Enforcer [${checks.selected}]`;
```

Comment:
If we allow selection of multiple items (not just all or one particular item), we'll need to find another way to display which are selected. We can even just show `Run Guideline Enforcer` without anything else and the user finds which items are selected when opening the selection menu, it could be enough in my opinion.

Reply:
No support for multi selection.
Also, to ensure the correct list display, the 1st item is always `All`, and this is the default selection.